canonical-websites/www.ubuntu.com
# Create basic template for User Report page

## Done

- Created the scaffolding for the new user report page ([copydoc](https://docs.google.com/document/d/153OkfZHM_eTfAoUZF3f38VMtiFzaa2txB9B2lvs8xXA/edit))
- Added the feature branch `user-report-page` based off beta

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/desktop/user-report
- Check that the page copy matches the [copydoc](https://docs.google.com/document/d/153OkfZHM_eTfAoUZF3f38VMtiFzaa2txB9B2lvs8xXA/edit) (though this is likely to change a lot, so it doesn't need to be 100% accurate for now)
- Check that it roughly matches the original [design](https://user-images.githubusercontent.com/23459065/43789362-af2f0aa8-9a67-11e8-9f19-3b6417980095.png)
- Check that the sticky nav changes active state on scroll

## Issue / Card

Fixes ubuntudesign/web-squad#763
